e2e/app.e2e.ts

```diff
@@ -84,6 +84,46 @@ describe('bterm launch', function() {
         .then(() => this.app.client.browserWindow.isFocused('.theme-browser > span:nth-child(2)'))
```

Comment:
The test will pass if the window has focus, not the second child of the .theme-browser.
I guess this is not the desired outcome.